Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apk add --no-cache \
   && apk add --no-cache --virtual=.builddeps \
         -X http://dl-cdn.alpinelinux.org/alpine/edge/community \
         git \
-        go \
+        go=1.8.4-r0 \
```

Comment:
Could you also make a card to remove this, that is blocked with the link to the go issue?

Reply:
Sure!